README.md

```diff
@@ -2,6 +2,10 @@
 
 This repository contains documents related to the overall governance of the NUnit organization as well as the individual projects under NUnit. It is currently under development.
 
+## Code of Conduct
+
+Please note that all NUnit projecst are released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in these projects you agree to abide by its terms.
```

Comment:
This wording is the pluralized wording recommended at http://contributor-covenant.org/, but I am fine with the change. I will hold off on making the change until others comment.